src/roswire/ros2/package.py

```diff
@@ -42,8 +46,13 @@ def _determine_paths(cls, app_instance: "AppInstance") -> List[str]:
             shell = app_instance.shell
             jsn = shell.check_output(_COMMAND_ROS2_PKG_PREFIXES, text=True)
         except dockerblade.exceptions.CalledProcessError:
-            logger.error("failed to obtain ROS2 package prefixes")
-            raise
+            # maybe python is python3, try that
```

Comment:
Ah. I think I see what's happening here. I think we can just use `python3` rather than `python` when we're working with ROS2.

Reply:
Done